content/F2015/F15-event-dropbox-coffee-chats.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug: dropbox-coffee-chats-f15
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
+
+Join us and several female engineers from Dropbox for coffee and snacks at William's in the plaza.
+
+Note: This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
```

Comment:
I really want to try and stay consistent with what we've done before. As far as I can tell we've never used relative path names for linking to the FAQ. I'm happy to discuss this further but bear in mind that these event posts are for documentation purposes so I don't think we need to worry about it.